src/test/java/com/training/web/tests/flipkart/PaymentPageTests.java

```diff
@@ -38,4 +38,16 @@ public void testEmiOptions(String bankName) {
         softAssertions.assertThat(paymentPage.getEmiSupport(bankName)).as("Bank details inaccurate")
                 .isEqualTo("No");
     }
+
+    @Test
+    public void testBackToTopFunctionality(){
+
+        paymentPage.scrollToPageEnd().clickBackToTop();
+
+        softAssertions.assertThat(paymentPage.getBackToTopIconVisibility()).as("Back to top element is still visible")
+                .isEqualTo(false);
+
+        softAssertions.assertThat(paymentPage.getHeaderVisibility()).as("Header element is not visible")
+                .isEqualTo(true);
```

Comment:
This can be written as - `softAssertions.assertThat(paymentPage.getHeaderVisibility()).as("Header element is not visible").isTrue()`